src/plugin/timezone/index.js

```diff
@@ -133,6 +133,10 @@ export default (o, c, d) => {
       // timestamp number || js Date || Day.js
       return d(input).tz(timezone)
     }
+    if (/Z$/i.test(input)) {
```

Comment:
Although I don't like this solution very much but I tried to be consistent with how `local` parses handles this:
https://github.com/iamkun/dayjs/blob/b5a1391011b247d08863d291542db5937b23b427/src/index.js#L60